mikeio1d/__init__.py

```diff
@@ -81,8 +80,8 @@ def open(file_name: str | Path, **kwargs) -> Res1D | Xns11:
     >>> res = mikeio1d.open("results.res1d")
     >>> res.nodes.read()
 
-    >>> xns = mikeio1d.open("cross_section.xns11")
-    >>> xns.xsections
+    >>> xs = mikeio1d.open("cross_section.xns11")
```

Comment:
I would move the open method to various.py or similar and not have actual implementations in init.